static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-content-vs-outbrain.js

```diff
@@ -24,12 +24,20 @@ define([
         variants: [
             {
                 id: 'paid-content',
-                test: function () {
+                // TODO: ask someone about this
```

Comment:
hey! I think the commercial team plans to revive that test eventually. cc @guardian/commercial-dev